src/Component/Button.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace NotificationChannels\WhatsApp\Component;
+
+abstract class Button extends Component
+{
+    protected string $sub_type = '';
+    protected int $index = 0;
+    protected array $parameters = [];
+
+    public function toArray(): array
+    {
+        return [
+            'type' => 'button',
+            'sub_type' => $this->sub_type, // quick_reply|url
+            'index' => '' . $this->index,
```

Comment:
Why are you casting the index from integer type to a string type?

Reply:
I am not sure if it is supposed to be a int or a string. In the example [below](https://developers.facebook.com/docs/whatsapp/cloud-api/reference/messages#template-messages) it is cast as a string. 
<img width="312" alt="Screenshot 2023-06-05 at 13 34 49" src="https://github.com/netflie/laravel-notification-whatsapp/assets/6515216/69c8989f-f499-47ea-8fa3-b0263f4dbcfc">

But the [documentation](https://developers.facebook.com/docs/whatsapp/cloud-api/reference/messages#components-object) does not specify.
<img width="815" alt="Screenshot 2023-06-05 at 13 34 40" src="https://github.com/netflie/laravel-notification-whatsapp/assets/6515216/47eb0e1c-46e5-485d-91c5-ff34f8e52992">

 I guess it might work as well as an int.